index.html

```diff
@@ -1787,6 +1876,172 @@ <h2>
+        </h3>
+        <p>
+          The [=file handler's=] <code><dfn data-dfn-for=
+          "file handler">action</dfn></code> member is a <a>string</a> that
+          represents a relative URL of the [=manifest/start_url=] origin that
+          is [=manifest/within scope=] of a [=Document/processed manifest=].
+          This URL will be navigated to in the [=launch event=] when handling a
+          file. See [=executing a file handler launch=] for more information.
+        </p>
+      </section>
+      <section>
+        <h3>
+          `name` member
+        </h3>
+        <p>
+          The [=file handler's=] <code><dfn data-dfn-for=
+          "file handler">name</dfn></code> member is a <a>string</a> that
+          represents a name to be provided to the user's operating system for
+          use with "Open with..." UX flows. This MAY be replaced by the user
+          agent for privacy & security reasons.
+        </p>
+      </section>
+      <section>
+        <h3>
+          `accept` member
+        </h3>
+        <p>
+          The [=file handler's=] <code><dfn data-dfn-for=
+          "file handler">accept</dfn></code> member is a <a>dictionary</a>
+          mapping MIME types to extensions. This ensures applications will work
+          on operating systems that only support one of the two (example:
+          Linux, which only supports MIME types). This also allows applications
+          to register custom file types. See steps to [=associate a file
+          handler=] for more information. (TODO fix this?)
+        </p>
+      </section>
+      <section>
+        <h3>
+          `icons` member
+        </h3>
+        <p>
+          The <a>file handler</a>'s <code><dfn data-dfn-for=
+          "file handler">icons</dfn></code> member lists images that serve as
+          iconic representations of file. This MAY be replaced by the user
+          agent for privacy & security reasons.
+        </p>
+      </section>
+      <section>
+        <h2>
+          Processing file handler items
+        </h2>
+        <p>
+          To <dfn>process a file handler item</dfn>, given [=ordered map=]
+          |item:ordered map|, [=URL=] |start_url:URL|, [=URL=] |scope:URL|, and
+          [=URL=] |manifest URL:URL|.
+        </p>
+        <ol class="algorithm">
+          <li>Return failure if it's the case that:
+            <ul>
+              <li>|item| is not [=ordered map=].
+              </li>
+              <li>|item|["action"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["action"] is not a [=string=].
+              </li>
+              <li>|item|["name"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["name"] is the empty string.
+              </li>
+              <li>|item|["accepts"] doesn't [=map/exist=].
+              </li>
+              <li>|item|["accepts"] is not a [=sequence=].
+              </li>
+              <li>|item|["accepts"] is an empty [=sequence=].
```

Comment:
Linkify empty

Reply:
Not sure there is something I can link to? only see list/empty

Reply:
Ah, okay, there's subtlety here. A "sequence" is a WebIDL idea. item and the stuff inside it are spec concepts, not IDL things. So this should be a list, not a sequence.